src/features/Classes/ClassDetailPage/index.jsx

```diff
@@ -80,6 +88,10 @@ const ClassDetailPage = () => {
 
   const handleEnrollStudentModal = () => setIsEnrollModalOpen(!isEnrollModalOpen);
 
+  const handleGradebookButton = () => {
```

Comment:
Please modify the classId to be decoded because if I tried with a class with special characters I can't get to class detail page